### PR TITLE
Fixed weird bug occuring on coexisting image and text components.

### PR DESCRIPTION
**Pull request by baransrc**

### CODE REVIEW

File: Source/src/components/ComponentImage.cpp [hunk 1]
```diff
@@ -127,6 +125,10 @@ void Hachiko::ComponentImage::Draw(ComponentTransform2D* transform, Program* pro
     program->BindUniformFloat4("img_color", render_color->ptr());
     ModuleTexture::Bind(img_to_draw? img_to_draw->GetImageId(): 0, static_cast<int>(Hachiko::ModuleProgram::TextureSlots::DIFFUSE));
 	glDrawElements(GL_TRIANGLES, 6, GL_UNSIGNED_INT, 0);
+
+    // Unbind square & deactivate program:
+    App->ui->UnbindSuare();
```
Comment (bernatgb) [inline Source/src/components/ComponentImage.cpp:130]: Shouldn't be named UnbindSquare?

**Review by bernatgb — APPROVED** [review]
Shouldn't UnbindSuare function be named UnbindSquare?

File: Source/src/components/ComponentImage.cpp (hunk 1 above)
Comment (baransrc) [reply Source/src/components/ComponentImage.cpp]: Thanks for the notice, it was not from my side, but I fixed it anyways 👍

**Review by Bernatmago — APPROVED** [review]

**Review by VBoschMatas — APPROVED** [review]
Yes. Me better apprub. Yes.

**Pull request merged**